scarlet/morphology.py

```diff
@@ -154,8 +151,9 @@ def __init__(self, frame, center):
 
     def get_model(self, *parameters):
         center = self.get_parameter(0, *parameters)
-        spec_bbox = Box((0,))
-        return self.psf(*center, bbox=spec_bbox @ self.bbox)[0]
+        box_center = np.mean(self.bbox.bounds[1:], axis=1)
+        offset = center - box_center
+        return self.psf.get_model(offset=offset)  # no "internal" PSF parameters here
```

Comment:
I agree in principle, but it's tied up with #177. I'd like to have the ability to change the boxes for all sources, not just for point sources, so it's better to treat them the same.

Reply:
That's fair. I just raised it because I found that it was extra bad for point sources. When shifting point sources to fit quasars, we found that the psf quickly ends up being chipped by the edge of the box. Feel free to resolve.